gtsfm/configs/scene_optimizer_unit_test_config.yaml

```diff
@@ -13,11 +13,13 @@ SceneOptimizer:
   two_view_estimator:
     _target_: gtsfm.two_view_estimator.TwoViewEstimator
     corr_metric_dist_threshold: 0.1
-    matcher:
-      _target_: gtsfm.frontend.matcher.twoway_matcher.TwoWayMatcher
+    matcher_verifier: 
+      _target_: gtsfm.frontend.matcher_verifier.combination_matcher_verifier.CombinationMatcherVerifier
+      matcher:
+        _target_: gtsfm.frontend.matcher.twoway_matcher.TwoWayMatcher
 
-    verifier:
-      _target_: gtsfm.frontend.verifier.degensac.Degensac
+      verifier:
+        _target_: gtsfm.frontend.verifier.ransac.Ransac
```

Comment:
switching to ransac for unit test as OpenCV's RANSAC has both e matrix and f matrix estimation